doc/dsdr/0001-creating-new-slices-at-the-simulation-level.md

```diff
@@ -0,0 +1,45 @@
+# Create new data slices at the simulation levels
+
+## Context
+
+The model projects future activity by creating multiple simulations of
+the future. We then run these simulation runs through a HDR Histogram
```

Comment:
change to "run these simulations" or "run these simulation results"

Reply:
fixed